modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -102,6 +102,7 @@ async fn await_consensus_to_catch_up(client: &Client, block_count: u64) -> anyho
 }
 
 #[tokio::test(flavor = "multi_thread")]
+//#[ignore]
```

Comment:
Leftover or on purpose?

Reply:
on purpose, being able to switch this on is very handy for development purposes

Reply:
I don't get why you move one test, and then disable bunch of others. Could you move and ... keep other ones as they were?

Reply:
Let's just say sometimes I'm this cat:
https://twitter.com/buitengebieden/status/1698416516457181500

Reply:
I still don't understand the ignores, seems like there has to be a better way.

Reply:
:confused: Can anyone explain for the slower members of the community such as myself why do we disable these innocent looking tests? :D

Reply:
If I disable some tests the whole suite will run faster. But then why not just run only the test I want? Because if the test run alone, I can't detect some types of flakiness. And this isn't hypothetical, right now these tests are flaky only if some of them run together.

So I disable most of tests and run a subset to be a fast but yet realistic scenario.